go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/onsi/ginkgo v1.14.0
 	github.com/onsi/gomega v1.10.1
 	github.com/pkg/errors v0.9.1
-	github.com/submariner-io/admiral v0.4.1
+	github.com/submariner-io/admiral v0.4.2-0.20200706185424-d52c64ccd7ba
```

Comment:
You had removed the admiral upgrade before - did you mean to add it back or did it sneak back in?